eclipse-scout-core/src/form/fields/wizard/WizardProgressField.ts

```diff
@@ -112,8 +114,8 @@ export class WizardProgressField extends FormField implements WizardProgressFiel
       this._updateStepClasses(step);
 
       // Inspector info
+      inspector.applyInfo(step, $step, this.session);
       if (this.session.inspector) {
-        inspector.applyInfo(step, $step);
         $step.attr('data-step-index', step.index);
```

Comment:
This could probably be safely added always without checking for `this.session.inspector`.